collector/kernel/process_handler.h

```diff
@@ -147,7 +89,6 @@ class ProcessHandler {
 
   struct ThreadGroupData {
     flowmill::kernel_collector::handles::tracked_process handle;
-    enum_traits<ProcPath>::array_map<std::string> proc_path;
     absl::flat_hash_map<u32, PerPidData> by_pid;
```

Comment:
Looks like `by_pid`, `by_tgid` and `last_check` fields will be left unused. Can they be removed?

Reply:
Sure